### ISSUE TITLE
distributing zip files of signatures, alone.

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#1320]
with greyhound #1226 #1238 and prefetch #1310, it would be really useful to be able to just distribute single zip files that contained signatures and nothing else.

ISTR @luizirber talking about this somewhere as distributing SBTs with no internal nodes, but I'm wondering if we even need JSON indices at all? I guess it wouldn't hurt, but I'm perfectly happy to not be required to load a big JSON at all.

**Comment by ctb** [comment]
did a little bit of playing around, and tried out the following command:
```
sourmash index test.sbt.zip podar-ref/*.fa.sig --sparseness=1 -k 31
```
which did indeed eliminate all internal nodes in the zip file, but of course did create a .sbt.json file. I'm pretty sure it also does the SBT creation in memory.

**Comment by ctb** [comment]
implemented in #1349.
